handbook/information-security/supplier-policy.md

```diff
@@ -0,0 +1,119 @@
+
+Our Supplier List tracks the suppliers we interact with. This is checked for
+completeness monthly by the [Information Security
+Officer](../README.md#contacts). As part of this check, we make sure all
+entities that have ever invoiced us are on the Supplier List.
+
+### Supplier Ownership
+
+The [Information Security Officer](../README.md#contacts) will assign a Supplier
+Owner to each Supplier in the Supplier List. These owners are tracked in the
+Supplier List. They are responsible for conducting the Vetting Process laid out
+below.
+
+### Vetting Process
+
+#### Vetting Requirement
+
+We organise suppliers into four categories:
+
+* Organisations without access to sensitive information.
+  * These do not require a security vetting.
+* Individuals and small organisations
+  * These are likely to require an NDA to be signed and/or a negotiation around
+    how to address vulnerabilities
+* Medium-sized organisation
+  * These are likely to require a review of their publicly available info sec
+    policies, a questionnaire where necessary and negotiations to address
+    vulnerabilities where necessary
+* Large organisation
+  * These are likely to require a review of their publicly available info sec
+    policies. Given they are significantly larger than us, it is usually not
+    feasible to require a questionnaire and/or negotiations. Therefore a
+    decision to collaborate or not will be made based solely on the publicly
+    available information.
+
+The Supplier Owner decides whether supplier vetting is required for an
+individual  supplier.
+
+If a supplier vetting is required, this is to be completed within two months of
+the supplier being added to the Supplier List.
+
+The Supplier Questionnaire can be found
+[here](https://goo.gl/forms/3J0BmcrOIhUxGP2x1) Responses to the Supplier
+Questionnaire can be found
+[here](https://docs.google.com/spreadsheets/d/1q4TB_HkJubEPoc4MjeColk0LG9AUr-6_2d_UvrnbY4g/edit?usp=sharing).
+
+Our standard NDA can be found
+[here](https://docs.google.com/document/d/1O_-nk5FLp-h5IycOM5Ntv0RxOcTWGWq7kfGZKqM8se4/edit?usp=sharing).
+
+#### Vetting Assessment
+
+In conducting the Vetting Assessment, the Supplier Owner assesses the available
+information and answers the following questions:
+
+1. Do we as an organisation sufficiently understand the risks of working with
+this supplier?
+
+2. Has the supplier adequately mitigated the risk of security incidents to
+minimise the probability of them occurring?
+
+3. Does the supplier's incident reporting policy allow us to effectively manage
+such incidents if they do occur?
+
+If the answer to all three questions is yes, the Supplier Owner accepts the
+supplier.
+
+#### Negotiating Mitigations
+
+If the answer to any of the three Vetting Assessment questions is no, the
+Supplier Owner may negotiate with the supplier to try to better mitigate the
+risks.
+
+Where such negotiations are entered, the final decision may be delayed by up to
```

Comment:
Negotiation sounds like the wrong word. We shouldn't be negotiating on risk. We should assess it and set clear milestones for progress along with a timeline to implementation.

If the milestones aren't reached and the risk is outside of our tolerance, then the supplier needs to be rejected.